.github/workflows/build.yml

```
@@ -51,3 +60,42 @@ jobs:
             massastation_${{ matrix.target }}_${{ matrix.arch }}.exe
           if-no-files-found: error
           retention-days: 7
+      - name: Upload Package
+        uses: actions/upload-artifact@v3
+        with:
+          name: massastation_${{ matrix.target }}_${{ matrix.arch }}_package
+          path: |
+            MassaStation.app
+            MassaStation.tar.xz
+          if-no-files-found: warn
+          retention-days: 1
```

why don't we use same value as others?

```suggestion
          if-no-files-found: error
          retention-days: 7
```

---

The `warn` is there only while I'm working on this part of the CI. It will be replaced by an error in next PRs.
And the retention-days is shorter because we need those artifacts only to build the installers. It's basically to save storage as much as possible